faq.md

```diff
@@ -5,64 +5,150 @@ layout: faq
+
+## How do I use this material?
+
+Many topics include slide decks and if the topic you are interested in has slides then start there.  These will introduce the topic and important concepts.
+
+Most of your learning will happen in the next step - the hands-on tutorials.  This is where you'll become familiar with using the Galaxy interface and experiment with different ways to use Galaxy and the tools in Galaxy.
+
+## Where can I run the hands-on tutorials?
+
+To run the hands-on tutorials you need a Galaxy server to run them on.
+
+Some topics have a [Docker](https://www.docker.com/) image that can be installed and run on participants' laptops.  These images contain Galaxy instances that include all tools and datasets used in a tutorial, as well as saved analyses and repeatable workflows that are relevant. You will need to [install Docker](https://docs.docker.com/install/).
+
+Many tutorials can be run on [public Galaxy servers](https://galaxyproject.org/public-galaxy-servers/).  These servers are available to anyone on the world wide web and some may have all the tools that are needed by a specific tutorial.
+
+If your organization/consortia/community has its own Galaxy server, then you may be able to run tutorials on that. Before you start you should confirm that all needed tools and reference genomes are available on your server.
+
+Finally, you can also run your tutorials on cloud-based infrastructures.  Galaxy is [available on many national research infrastructures](https://galaxyproject.org/galaxy-services/) such as [Jetstream](https://galaxyproject.org/cloud/jetstream/) (United States), [GenAP](https://www.genap.ca/) (Canada), [GVL](https://launch.genome.edu.au/launch) (Australia), [CLIMB](http://www.climb.ac.uk/) (United Kingdom), and more.  These instances are typically easy to launch, and easy to shut down when you are done.
+
+If you are already familiar with, and have an account on [Amazon Web Services](https://aws.amazon.com/) then you can also launch a Galaxy server there using [CloudLaunch](https://launch.usegalaxy.org/).
+
+## How can I get help?
+
+If you have questions about this training material, you can reach us using the [Gitter chat](https://gitter.im/Galaxy-Training-Network/Lobby).  You'll need a [GitHub](https://github.com/) or [Twitter](https://twitter.com/) account to post questions.  If you have questions about Galaxy outside the context of training, see the [Galaxy Support page](https://galaxyproject.org/support/).
+
+# For Instructors
+
+This material can also be used to teach Galaxy and data analysis in a group setting to students and researchers.
+
+## Where do I start?
+
+Spend some time exploring the different tutorials and the different resources that are available. Become familiar with the structure of the tutorials and think about how you might use them in your teaching.
+
+## What Galaxy instance should I use for my training?
```

Comment:
To avoid redundancy, we could redirect to the previous section for user. Will be easier to keep up to date. What do you think?